schemas/preferences.sublime-settings.json

```diff
@@ -11,7 +11,19 @@
         "color_scheme": {
           "markdownDescription": "Sets the colors used within the text area.",
           "type": "string",
-          "pattern": "^(?:(.+\\.((sublime|hidden)-color-scheme|tmTheme)|auto)$",
+          "pattern": "^(.+\\.((sublime|hidden)-color-scheme|tmTheme)|auto)",
+          "default": "Mariana.sublime-color-scheme"
```

Comment:
The default is "auto" I believe.

Reply:
The default is Mariana.sublime-color-scheme.